test-suite/build.gradle

```diff
@@ -25,7 +25,8 @@ dependencies {
     testImplementation "io.micronaut:micronaut-http-server-netty:$micronautVersion"
     testImplementation "io.micronaut:micronaut-http-client:$micronautVersion"
     testImplementation project(":microstream")
-
+    testImplementation "io.micronaut.cache:micronaut-cache-core"
```

Comment:
Is this not `libs.micronaut.cache.core` and so on?

Reply:
done via https://github.com/micronaut-projects/micronaut-microstream/pull/23/commits/b8f8af8f9e6bfbba7fa73af20a84b7bd3ccaf5d1